src/pages/components/accordion/code.mdx

```diff
@@ -1,22 +1,177 @@
 ---
 title: Accordion
-description: The accordion component delivers large amounts of content in a small space through progressive disclosure. The user gets key details about the underlying content and can choose to expand that content.
-tabs: ["Code", "Usage", "Style", "Accessibility"]
+description:
+  The accordion component delivers large amounts of content in a small space
+  through progressive disclosure. The user gets key details about the underlying
+  content and can choose to expand that content.
+tabs: ['Code', 'Usage', 'Style', 'Accessibility']
 ---
 
+import usage from './snippets/usage';
+import skeleton from './snippets/skeleton';
+
+<PageDescription>
+
+Learn how to build an accordion in React.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Usage</AnchorLink>
+<AnchorLink>Component API</AnchorLink>
+<AnchorLink>FAQ</AnchorLink>
+
+</AnchorLinks>
+
+## Usage
+
+You can build an accordion using a combination of the `Accordion` and
+`AccordionItem` components. The `Accordion` components acceps a list of
+`AccordionItem` components as `children` and each `AccordionItem` is responsible
+for displaying the accordion's heading and panel content.
+
+You can configure the accordion item's heading using the `title` prop.
+Everything you pass in as a child of `AccordionItem` will be rendered in the
+accordion's panel.
+
+<ComponentDemo links={{}}>{usage}</ComponentDemo>
```

Comment:
Just wondering why we don't want to have the knob container here to show the interaction of the props? I thought that was kind of the point of the knobs in the demo? So users can see the changes in action.

Reply:
@jnm2377 great question, I think knobs, in general, have presented a challenge for us on the documentation side. Tried to share a little bit more about this over in https://github.com/carbon-design-system/carbon/issues/5640#issuecomment-601255445

Some of this is very much so storybook specific (like copy/paste-ability), other bits are kind of a consequence of the format. The biggest shortcoming is that we would like to offer inline snippets/explanations for props and how they should be used either in isolation or in conjunction with other props or scenarios.